### DISCUSSION TITLE
#47 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#27] [General]
Agenda:

Community updates
Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by bambalados** [comment]
**CTN** 

- CTN17 - 主题：Paradigm CTF - 09.08

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

9个PR #822 ～ #830

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《使用Foundry加速智能合约开发》 by Alan
  - https://www.youtube.com/watch?v=EXYeltwvftw
  - https://www.youtube.com/watch?v=tIsRR-ekmgU

- 《GMX项目分享》by Comcat
  - https://www.youtube.com/watch?v=_KyDkEu0sYs
  - https://www.youtube.com/watch?v=7wTPIUw6iGo

**Comment by stephanieeee-ho** [comment]
### ECN
日常的翻译和周报输出

**Comment by Stamp9** [comment]
## 原语里弄

密码学知识普及系列 - 共建门限签名知识库
https://shoutout.wix.com/so/32OC8VFhC

**Comment by luckyyang** [comment]
### Rebase
- web3 daily 更新 14 期 https://rebase.network/posts/category/web3daily
- 招聘: 发布 7 个职位 https://github.com/rebase-network/who-is-hiring/issues
- 活动: 支持 2 个活动 https://github.com/rebase-network/event-pool/issues

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译正常进行中
